evals/data.py

```diff
@@ -104,10 +106,20 @@ def wrapper(*args, **kwargs):
             logger.debug(f"Loading from file cache: {pkl_path}")
```

Comment:
we can change this to INFO so that it's easier to tell when we're loading from file cache